pkgs/applications/networking/cluster/k3s/default.nix

```
@@ -5,9 +5,11 @@
 , iptables
 , iproute2
 , bridge-utils
+, btrfs-progs
 , conntrack-tools
-, buildGoPackage
+, buildGoModule
```

```suggestion
, buildGo117Module
```
So you don't have to change top-level.